pkgs/by-name/fs/fsnav/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  freetype,
+  libpng,
+  libjpeg,
+  libGL,
+  libGLU,
+  freeglut,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fsnav";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "jtsiomb";
+    repo = "fsnav";
+    rev = "v${version}";
+    hash = "sha256-Bt1QRqtJkVFR1uMzmB3OUyqyzUyJdQyQdbMOfMyWJOc=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    freetype
+    libpng
+    libjpeg
+    libGL
+    libGLU
+    freeglut
+  ];
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
```

this doesn't fix compilation with Clang[^1] (due to absence of gcc and g++, which are set by default), so let's make use of envvars
```suggestion
  preBuild = ''
    makeFlagsArray+=(
      "PREFIX=$out"
      "CC=$CC"
      "CXX=$CXX"
    )
  '';
```
[^1]: `nix-build -E 'with import path/to/nixpkgs {}; fsnav.override {stdenv=clangStdenv;}'`